pkg/service/sync.go

```diff
@@ -97,6 +102,11 @@ func (s *SyncService) SyncRewardHistory(ctx context.Context, exchange types.Exch
 
 func (s *SyncService) SyncDepositHistory(ctx context.Context, exchange types.Exchange, startTime time.Time) error {
 	log.Infof("syncing %s deposit records...", exchange.Name())
+	if util.IsPaperTrade() {
```

Comment:
I need to think about the design, how about just return empty slice from the binance implementation if paper trade is on?

Reply:
```return nil``` works now
In addition, the default return of SyncDepositHistory is nil

Reply:
sounds good, thanks!